controllers/provider-packet/pkg/apis/config/helper/helper.go

```diff
@@ -18,17 +18,28 @@ import (
 	"fmt"
 
 	"github.com/gardener/gardener-extensions/controllers/provider-packet/pkg/apis/config"
+	api "github.com/gardener/gardener-extensions/controllers/provider-packet/pkg/apis/packet"
```

Comment:
```suggestion
	apispacket "github.com/gardener/gardener-extensions/controllers/provider-packet/pkg/apis/packet"
```

Reply:
I intentionally used a generic label here. This allows copying the code among the providers without changing every the label, just by changing the import.